samples/connectedhomeip/light_bulb/README.rst

```diff
@@ -0,0 +1,202 @@
+
+.. _chip_light_bulb_network_mode:
+
+Remote testing in a network
+===========================
+
+.. include:: ../lock/README.rst
+    :start-after: chip_door_lock_sample_remote_testing_start
+    :end-before: chip_door_lock_sample_remote_testing_end
+
+.. _chip_light_bulb_sample_test_mode:
+
+Remote testing using test mode
+==============================
+
+.. include:: ../lock/README.rst
+    :start-after: chip_door_lock_sample_test_mode_start
+    :end-before: chip_door_lock_sample_test_mode_end
+
+Using the test mode allows you to control the light bulb remotely without using a smartphone compatible with Android.
+The light bulb device programmed with this sample can be used with the light switch device programmed with the `chip_light_switch_sample` sample to create a simplified Thread network.
+
+User interface
+**************
+
+.. include:: ../lock/README.rst
+    :start-after: chip_door_lock_sample_led1_start
+    :end-before: chip_door_lock_sample_led1_end
+
+LED 2:
+    Shows the state of the light bulb.
+    The following states are possible:
+
+    * Solid On - The light bulb is on.
+    * Off - The light bulb is off.
+
+Button 1:
+    Initiates the factory reset of the device.
+
+Button 2:
+    Changes the light bulb state to the opposite one.
+
+Button 3:
+    Starts the Thread networking in the :ref:`test mode <chip_light_bulb_sample_test_mode>` using the default configuration.
+
+    When running application in light switch test mode it also starts publishing the light bulb service messages for a predefined period of time to advertise the light bulb device IP address to the light switch device (if used).
+    If for some reason the light switch device is not able to receive messages during this predefined period of time, you can restart publishing service by pressing this button again.
+
+Button 4:
+    Starts the the NFC tag emulation, enables Bluetooth LE advertising for the predefined period of time and makes the device discoverable over Bluetooth LE.
+    This button is used during the :ref:`commissioning procedure <chip_light_bulb_sample_remote_control_commissioning>`.
+
+SEGGER J-Link USB port:
+    Used for getting logs from the device or communicating with it through the command-line interface.
+
+NFC port with antenna attached:
+    Optionally used for obtaining the commissioning information from the CHIP device to start the :ref:`commissioning procedure <chip_light_bulb_sample_remote_control_commissioning>`.
+
+Building and running
+********************
+
+.. |sample path| replace:: :file:`samples/connectedhomeip/light_bulb`
+
+.. include:: /includes/build_and_run.txt
+
+.. _chip_light_bulb_building_with_light_switch_overlay:
+
+Building for Light switch test mode
+===================================
+
+To build the sample with configuration that allows testing with light switch sample, run the following command:
```

Comment:
Should we mention somewhere, what is the test mode? I.e. that Commissioning process is bypassed, and keys hardcoded.

Reply:
I think this is already covered by the reused lines from the door lock sample:

Remote testing using test mode
==============================

.. include:: ../lock/README.rst
    :start-after: chip_door_lock_sample_test_mode_start
    :end-before: chip_door_lock_sample_test_mode_end

Which expands to:

Alternatively to the commissioning procedure, you can use the test mode, which allows to join a Thread network with default static parameters and static cryptographic keys. Use Button 3 to enable this mode after building and running the sample.